docker-compose.yaml

```diff
@@ -188,6 +188,8 @@ services:
         expose:
             - ${RABBITMQ_PORT}
             # - 15672 management port
+        ports:
+            - 15672:15672
```

Comment:
shouldn't those be set by the environment?

Reply:
oops, got through by mistake